packages/fluent-theme/src/components/Theme.module.css

```diff
@@ -6,15 +6,16 @@
   --webchat-colorNeutralForeground2: var(--colorNeutralForeground2, #424242);
   --webchat-colorNeutralForeground4: var(--colorNeutralForeground4, #707070);
 
+  --webchat-colorNeutralForeground2BrandHover: var(--colorNeutralForeground2BrandHover, #02729c);
```

Comment:
Should we have a `@media` query so we will use dark theme colors if the Fluent defaults is not loaded?